docker-compose/versions/camunda-8.7/.env

```diff
@@ -39,3 +39,12 @@ RESOURCE_AUTHORIZATIONS_ENABLED=false
 #  ZEEBE_AUTHENTICATION_MODE=identity
 #
 MULTI_TENANCY_ENABLED=false
+
+# Document store credentials
+DOCUMENT_DEFAULT_STORE_ID=...
+AWS_ACCESS_KEY_ID=...
+AWS_SECRET_ACCESS_KEY=...
+AWS_REGION=..
+DOCUMENT_STORE_AWS_BUCKET=...
+DOCUMENT_STORE_AWS_BUCKET_PATH=...
+DOCUMENT_STORE_AWS_CLASS=...
```

Comment:
I wonder if we should comment these out. I'm not sure if anything would be affected by these values being ...